en/micro-integrator/docs/references/connectors/servicenow-connector/servicenow-connector-example.md

```diff
@@ -0,0 +1,250 @@
+
+* `/readRecord `: It reads the detailed information about the created incident record in the incident table.
+    <img src="/assets/img/connectors/serviceNow.png" title="Adding a Rest API" width="800" alt="Adding a Rest API"/>
+
+## Configure the connector in WSO2 Integration Studio
+
+Follow these steps to set up the ESB Solution Project and the Connector Exporter Project. 
+
+{!references/connectors/importing-connector-to-integration-studio.md!} 
+
+1. First let's create postRecord sequence and ReadRecord sequences. Right click on the created ESB Solution Project and select, -> **New** -> **Sequence** to create the Sequence. 
+    <img src="/assets/img/connectors/add-sequence.png" title="Adding a Sequence" width="800" alt="Adding a Sequence"/>
+
+2. Provide the Sequence name as PostRecord. You can go to the source view of the xml configuration file of the API and copy the following configuration. 
+    ```
+      <?xml version="1.0" encoding="UTF-8"?>
+      <sequence name="PostRecord" trace="disable" xmlns="http://ws.apache.org/ns/synapse">
+      <servicenow.init>
+          <serviceNowInstanceURL>https://dev55707.service-now.com</serviceNowInstanceURL>
+          <username>admin</username>
+          <password>Diazo123@</password>
+      </servicenow.init>
+      <servicenow.postRecord>
+          <tableName>incident</tableName>
+          <sysparmDisplayValue>true</sysparmDisplayValue>
+          <sysparmFields>short_description,number,sys_id</sysparmFields>
+          <sysparmView>short_description,number,sys_id</sysparmView>
+          <sysparmInputDisplayValue>true</sysparmInputDisplayValue>
+          <number>34</number>
+          <shortDescription>{$ctx:shortDescription}</shortDescription>
+          <active>true</active>
+          <approval>owner</approval>
+          <category>inquiry</category>
+          <contactType>{$ctx:contactType}</contactType>
+      </servicenow.postRecord>
+      <property expression="json-eval($.result.sys_id)" name="sysId" scope="default" type="STRING"/>
+      </sequence>
+    ```
+3. Create the ReadRecord sequence as below. 
+  ```
+    <?xml version="1.0" encoding="UTF-8"?>
+    <sequence name="ReadRecord" trace="disable" xmlns="http://ws.apache.org/ns/synapse">
+        <servicenow.init>
+            <serviceNowInstanceURL>https://dev55707.service-now.com</serviceNowInstanceURL>
+            <username>admin</username>
+            <password>Diazo123@</password>
+        </servicenow.init>
+        <servicenow.getRecordById>
+            <sysId>{$ctx:sysId}</sysId>
+            <tableName>incident</tableName>
+        </servicenow.getRecordById>
+    </sequence>
+  ```
+  4. Now right click on the created ESB Solution Project and select, -> **New** -> **Rest API** to create the REST API. 
+    <img src="/assets/img/connectors/adding-an-api.png" title="Adding a Rest API" width="800" alt="Adding a Rest API"/>
```

Comment:
Do we need this image? @samgnaniah idea is having a lot of images in the doc make it confusing to the user.

Reply:
Removed